shell/platform/fuchsia/flutter/platform_view.h

```diff
@@ -109,6 +109,10 @@ class PlatformView final : public flutter::PlatformView,
       fit::function<void(
           fml::RefPtr<flutter::PlatformMessage> /* message */)> /* handler */>
       platform_message_handlers_;
+  // These are the channels that aren't registered and have been notified as
+  // such. Notifying via logs multiple times results in log-spam. See:
+  // https://github.com/flutter/flutter/issues/55966
+  std::set<std::string /* channel */> unregistered_channels_;
```

Comment:
can we have multiple PlatformViews over the lifetime of a process? do we want this to be a static?

Reply:
There's only one platform view per engine so I think that should be fine. Given that what we are seeing now, it this message being printed at error levels every time we receive a platform message on the same channel, I changed it to be INFO and made it so that we log a channel once per engine. So I think this should reduce the problem by a lot. If we notice that we are still seeing spammy behavior making this a static is something we could consider.